tools/points-service/main.go

```diff
@@ -81,7 +80,7 @@ var (
 		Name:    "ethereum-rpc-url",
 		Usage:   "URL of the Ethereum RPC server",
 		EnvVars: []string{"POINTS_ETH_RPC_URL"},
-		Value:   "https://eth-holesky.g.alchemy.com/v2/0DDo7YeieNEucZX3jieFfzmzOCGTKAgp",
+		Value:   "https://eth-mainnet.g.alchemy.com/v2/0DDo7YeieNEucZX3jieFfzmzOCGTKAgp",
```

Comment:
The API key should not be here. Instead, there should be some public node as default.